src/agoradatatools/etl/transform/biodomain_info.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
+
+
+def transform_biodomain_info(datasets: dict) -> pd.DataFrame:
+    """Takes dictionary of dataset DataFrames, extracts the genes_biodomains
+    DataFrame, gets a unique list of biodomain names, and outputs the list as
+    a single-column DataFrame with column "name".
+
+    Args:
+        datasets (dict[str, pd.DataFrame]): dictionary of dataset names mapped to their DataFrame
+
+    Returns:
+        pd.DataFrame: 1-column DataFrame with column "name"
+    """
+    genes_biodomains = datasets["genes_biodomains"]
+    biodomain_info = (
+        genes_biodomains["biodomain"]
+        .dropna()
+        .drop_duplicates()
+        .reset_index()
+        .drop(columns="index")
+        .rename(columns={"biodomain": "name"})
```

Comment:
I'm ok with this here, but just wanted to understand why we aren't using the `column_rename` or `agora_rename` functionality.